mamba.xcodeproj/project.pbxproj

```diff
@@ -2376,6 +2460,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				BUILD_LIBRARY_FOR_DISTRIBUTION = YES;
+				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = NO;
```

Comment:
Any idea why this is added?

I'm not familiar with this build setting and I notice it doesn't exist in the equivalent `develop_1.x` branch.

Quick scan of Google shows up this SO thread: https://stackoverflow.com/questions/24103169/swift-compiler-error-non-modular-header-inside-framework-module

Actually... Seems like the default for this setting is already `NO`, so perhaps this change isn't even needed? _(Same for the change at line 2492)_.

Reply:
@theRealRobG Not sure why this setting was added but perhaps something that comes from building with Xcode 16? I did not deliberately add it for this work.

If it is the default, I'm content with leaving it as is if that's alright?